pom.xml

```diff
@@ -43,14 +43,13 @@
 
         <shadeBase>io.prestosql.hive.\$internal</shadeBase>
 
-        <dep.hive.version>2.3.4</dep.hive.version>
+        <dep.hive.version>3.0.0</dep.hive.version>
```

Comment:
I expect that Hive 3.0 is not an interesting version. Everybody who upgrades to Hive 3 will probably upgrade to 3.1.
I am ok with upgrading to 3.0 if
- we later upgrade to 3.1
- or we realize that my expectations do not match real world

Reply:
nvrm; https://github.com/prestosql/presto/pull/198

Reply:
I agree, it's not interesting. This is to update the Parquet version since the one in `2.3.4` is broken.